src/openms/include/OpenMS/METADATA/PeptideHit.h

```diff
@@ -283,6 +283,8 @@ namespace OpenMS
 
     /// returns the fragment annotations
     std::vector<PeptideHit::PeakAnnotation> getPeakAnnotations() const;
+    const std::vector<PeptideHit::PeakAnnotation>& getPeakAnnotationsRef() const;
```

Comment:
Please without Ref

Reply:
No ret val overload possible. Then I have to check all other occurrences if they actually need a copy or not.